cluster/src/main/java/com/hartwig/pipeline/turquoise/Turquoise.java

```diff
@@ -24,6 +24,11 @@ public void publishStarted() {
             public void publishComplete(final String status) {
                 // noop
             }
+
+            @Override
+            public void close() {
+                publisher.shutdown();
+            }
```

Comment:
When calling the static factory to create this instance, you still pass along an actual `com.google.cloud.pubsub.v1.Publisher` as argument. So even though this implementation doesn't do anything in terms of publishing, I think it is still responsible for correctly shutting down the actual publisher it composes.

Reply:
Hrm OK, not going to push it too much but this seems like it could cause trouble if that publisher is shared at all. You've thought about it so let's leave it at that.